### PR TITLE
Commerce 1731 Promo Price fixes

### PR DESCRIPTION
**Pull request by alecsloan**

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by ethanbustad** [comment]
@alecsloan I think this actually is a failure that needs to be fixed in this PR, fyi

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

File: commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderItemLocalServiceImpl.java [hunk 1]
```diff
@@ -133,6 +134,7 @@ public CommerceOrderItem addCommerceOrderItem(
 		commerceOrderItem.setShippedQuantity(shippedQuantity);
 		commerceOrderItem.setJson(json);
 		commerceOrderItem.setUnitPrice(unitPrice.getPrice());
+		commerceOrderItem.setPromoPrice(promoPrice.getPrice());
```
Comment (marco-leo) [inline commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderItemLocalServiceImpl.java]: check for null

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

File: commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/CommercePendingOrderItemClayTable.java [hunk 2]
```diff
@@ -178,54 +174,36 @@ public String getId() {
...
-						commerceProductPrice.getDiscountValue();
-
-					if (discountValue != null) {
-						CommerceMoney discountAmount =
-							discountValue.getDiscountAmount();
-
-						discount = discountAmount.format(
-							themeDisplay.getLocale());
-					}
-				}
```
Comment (marco-leo) [inline commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/CommercePendingOrderItemClayTable.java]: revert this logic please

**Comment by marco-leo** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Pull request merged**